src/components/posts/PostsPage.js

```diff
@@ -42,6 +42,10 @@ const PostsPage = ({ getUserId }) => {
 
       .post("/post", post)
       .then(res => setPostResponse(res.data))
+      .then(() => {
+        const inputForm = document.getElementById('user-form-input')
+        inputForm.value = ''
```

Comment:
What you have here with `<TextField />` is an uncontrolled component - not recommended. You want your `PostForm` component to control this field. Instead of using the DOM to get the element, add a `useState` to hold the value being entered in the TextField and give that TextField component a `value` prop.